tests/test_browser.py

```diff
@@ -2582,3 +2582,93 @@ def test_modularize(self):
         ''' % code)
         self.run_browser('a.html', '...', '/report_result?0')
 
+  # Test that the emscripten_ atomics api functions work.
```

Comment:
do these tests pass with a warning if there is no pthreads support in the browser?